lib/mixlib/install/product.rb

```diff
@@ -193,8 +193,13 @@ def lookup(key, version = :latest)
 
   product "automate" do
     product_name "Chef Automate"
-    package_name "delivery"
-    ctl_command "delivery-ctl"
+    # Delivery backward compatibility
+    package_name do |v|
+      v < version_for("1.0.0") ? "delivery" : "automate"
```

Comment:
@PrajaktaPurohit @Sgtpluck @mhclin What will be the first Automate version number we ship a package named `automate`?